javascript/javascript2/week1/script.js

```diff
@@ -0,0 +1,27 @@
+//find shortest word
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+let shortestWordLength= danishWords[0].length;
+danishWords.forEach(shortestWordinArray)
+function shortestWordinArray(wordsArray){   
+    if(wordsArray.length < shortestWordLength){
+        shortestWordLength=wordsArray.length;
+        console.log('shortest word "'+wordsArray +'" length :'+shortestWordLength);   
+    } 
+}
+//Find and count the Danish letters
+const danishString = "Jeg har en blå bil";
+findDanishAlphabets(danishString); // returns {total: 1, å: 1}
+
+const danishString2 = "Blå grød med røde bær";
+findDanishAlphabets(danishString2); // returns {total: 4, æ: 1, ø: 2, å: 1}
+
+function findDanishAlphabets(Sentance){
+    let danishSentance=Sentance.toLowerCase();
+    let splitedSentance = danishSentance.split('');
+    const letter_æ=splitedSentance.filter(letter=>letter=="æ");
```

Comment:
This approach iterates the array 3 times. It can be iterated once if you just map over it and keep your counters outside of the map function.